exercises/concept-wip/crystal-hunter/.docs/hints.md

```diff
@@ -0,0 +1,25 @@
+# Hints
+
+## General
+
+- Don't worry about how the arguments are _derived_, focus on combining the arguments to return the intended result.
```

Comment:
What does this mean? Are the values derived in some confusing way?

Reply:
Tbh I just copied that from the exercise I borrowed it from, not 100% sure personally, but I think it is about that you don't have to think about the logic if the character is touching the crystal since that logic is already given to you.